include/swift/AST/CASTBridging.h

```diff
@@ -31,6 +31,69 @@
 #endif
 
 SWIFT_BEGIN_NULLABILITY_ANNOTATIONS
+
+typedef struct BridgedDeclContext {
+  void *_Nonnull raw;
+} BridgedDeclContext;
```

Comment:
Why can’t we use the `AST_BRIDGING_WRAPPER_NONNULL` macro for `BridgedDeclContext`?

Reply:
Good catch, this was needed in an earlier revision to avoid defining `bridged(DeclContext *)`, but I don't think it actually causes any issues any more, I'll change it